operator/pom.xml

```diff
@@ -86,7 +86,7 @@
                     <images>
                         <image>
                             <alias>watcher</alias>
-                            <name>docker.io/streamzi/cef-watcher</name>
```

Comment:
cef stands for cloud-event-flow, should it be changed to ef-watcher?

Reply:
@hhiden I'd think we go with `evenflow`, as full name. but `ef` is OK for me as well

Reply:
So eventflow-operator ?

Reply:
Agreed eventflow is better